src/modules/Builder/components/ConditionRow/Condition/SwitchCondition/SwitchCondition.tsx

```diff
@@ -65,8 +66,34 @@ export const SwitchCondition = ({
       payloadName,
       itemsName,
       itemName,
+      dateValueName,
     ],
   });
+
+  const parsedMinDateValue = minDateValue && new Date(minDateValue);
+  const parsedMaxDateValue = maxDateValue && new Date(maxDateValue);
+  const parsedDateValue = dateValue && new Date(dateValue);
```

Comment:
I think these values can be omitted

Reply:
yes, I initially created those parsed dates because I was using value in input, I thought it would be necessary to keep updating the parsed date, but it worked without it.